locust/webui/.yarnrc

```diff
@@ -0,0 +1 @@
+network-timeout 500000
```

Comment:
Hey yeah - genuinely I started working on this on my old laptop (Macbook, intel, 16gb) and I needed this (despite having a 1gb down wired connection) otherwise I was getting the same timeouts locally that you'd see on an underspec'd docker container

On my shiny new laptop (Macbook, M3, 36gb) I doubt I'd need it - so I suppose it's a case of whether we should add it for lethargic build machines in the local build or not 🤔

Reply:
Haha I've never seen yarn timeouts on my local before but I don't think having a longer timeout will hurt anyone :+1: